iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/GeneratorModification.java

```diff
@@ -84,6 +84,14 @@ private void setTargetPWithinBoundaries(Generator g, double targetP, boolean ski
         g.setTargetP(Math.min(g.getMaxP(), Math.max(g.getMinP(), targetP)));
     }
 
+    public Modifs getModifs() {
+        return modifs;
+    }
+
+    public String getGeneratorId() {
+        return generatorId;
+    }
+
     public static class Modifs {
```

Comment:
How does it work if for a generator I want to apply a change only on `targetQ` and `targetV`?

Reply:
The other values will not be set for our application in OpenReac. Thus only targetV and targetQ won't be null.